spec/factories/user.rb

```diff
@@ -0,0 +1,10 @@
+require 'faker'
+
+FactoryGirl.define do 
+  factory :user do |f|
+    f.name { Faker::Name.name }
+    f.email "bla@bla101.com"
+    f.password              "password"
+    f.password_confirmation "password"
```

Comment:
Faker can provide example emails and passwords too. You can find everything it gives you in the documentation.